libshaderc_spvc/CMakeLists.txt

```diff
@@ -83,22 +76,9 @@ if(SHADERC_ENABLE_INSTALL)
     ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
 endif(SHADERC_ENABLE_INSTALL)
 
-find_package(Threads)
-set(SPVC_LIBS
-  ${CMAKE_THREAD_LIBS_INIT}
-  SPIRV-Tools
-  SPIRV-Tools-opt
-  spirv-cross-glsl
-  spirv-cross-hlsl
-  spirv-cross-msl
-)
-
-target_link_libraries(shaderc_spvc PRIVATE ${SPVC_LIBS})
-target_link_libraries(shaderc_spvc_shared PRIVATE ${SPVC_LIBS})
-
 shaderc_add_tests(
   TEST_PREFIX shaderc
-  LINK_LIBS shaderc_spvc
+  LINK_LIBS shaderc_spvc ${SPVC_LIBS}
```

Comment:
Where is ${SPVC_LIBS} set? Looks like you removed it.

Reply:
I messed up the merge, re-ran it and force-pushed it up.